exchanges/alert/alert.go

```diff
@@ -5,6 +5,19 @@ import (
 	"sync/atomic"
 )
 
+// pool is a silent shared pool between all notice instances for alerting
+// external routines waiting on a state change.
+var pool = sync.Pool{New: func() interface{} { return make(chan bool) }}
+
+const (
+	inactive = uint32(iota)
+	active
+	alerting
+
+	dataToActuatorDefaultBuffer = 1
+	preAllocCommsDefaultBuffer  = 5
```

Comment:
Should these be config variables?

Reply:
`dataToActuatorDefaultBuffer` probably not, `preAllocCommsDefaultBuffer`; maybe?

Reply:
@gloriousCode preAllocCommsDefaultBuffer has been linked to engine settings and readme has been added for usage.